### DISCUSSION TITLE
有人认识Random Play店长吗

### DISCUSSION THREAD

**Discussion by Leo2126** [share121/inter-knot#2853] [其他]
有人认识Random Play店长吗？
最近几天我看这位店长从店里跑出来找人（忘了叫什么名字），还念叨着什么“好感”、“任务”
请帮我转告他，他要找的人正在Random Play前台和小18一起宣传

**Comment by btazsq** [comment]
我好像也看到了，和小18一起宣传的人似乎是安比。看起来，狡兔屋最近的经济状况很紧张啊，一直在给人打工。

**Reply by MTfriday** [reply]
安比？是那个天天在M记吃汉堡的女孩吗

**Comment by HoshimiMiyabi120** [comment]
嗯……店长的话，我倒是很熟识

**Reply by wpapkcn** [reply]
怎么哪都能见到雅科长，冲浪强度也太高了吧
